db/migrate/20161011151353_devise_create_users.rb

```diff
@@ -8,7 +10,7 @@ def change
       ## Recoverable
       t.string   :reset_password_token
       t.datetime :reset_password_sent_at
-      t.boolean  :allow_password_change, default: false
+      t.boolean  :allow_password_change, default: false, null: false
```

Comment:
Do we want to edit an existing migration? Is it safe? Or would it be better to add a new migration with this change?

Reply:
Even if we add a new one, the linter issue will appear the same. Anyway, I think it's safe.